handlers/propertyInspectionReports.js

```diff
@@ -27,5 +33,31 @@ function propertyInspectionReports(request, reply) {
 
 }
 
-module.exports = propertyInspectionReports;
+function postHandler(request, reply) {
+  var payload = request.payload;
+  var userId = 6; //Temporary hardcoded - userid currently can't be passed via params
```

Comment:
Ahh forgot to ask you about that! Yeah I wasn't sure how to retrieve the userId at the time as it was different from the previous implementations. I was in a rush so I had overlooked it. Thanks again for the reminder! I'll try make sure it doesn't happen again (and wait to ask before doing a PR).

I saw the fix using auth.artifacts, curious when this is available for use?